src/screens/Assistant/SelectFavouritesBottomSheet.tsx

```diff
@@ -61,6 +74,7 @@ const SelectableFavouriteDeparture = ({
 
         <ThemeText type="body__secondary" style={styles.secondaryText}>
           {t(SelectFavouriteDeparturesText.departures.from)} {departureStation}
+          {departureQuay && ' ' + departureQuay}
```

Comment:
Maybe calculate this inside the Text file itself? For eg https://github.com/AtB-AS/mittatb-app/blob/master/src/translations/screens/Tickets.ts#L171